src/logic/scriptExternals/Stubs.cpp

```diff
@@ -415,13 +415,6 @@ void ::Logic::ScriptExternals::registerStubs(Daedalus::DaedalusVM& vm, bool verb
         vm.setReturn(0);
     });
 
-    vm.registerExternalFunction("npc_hasequippedmeleeweapon", [=](Daedalus::DaedalusVM& vm) {
-        if(verbose) LogInfo() << "npc_hasequippedmeleeweapon";
-        uint32_t arr_self;
-        int32_t self = vm.popVar(arr_self); if(verbose) LogInfo() << "self: " << self;
-        vm.setReturn(0);
-    });
-
```

Comment:
Could you re-add the script function in the Stubs.cpp?
It's better to keep the prototype in the stubs just in case something goes wrong or for quick lookup of the methods interface.